project/templates/app/mobi/app/home.html

```diff
@@ -3,7 +3,7 @@
 {% load cache mama_inclusion_tags livechat_tags survey_inclusion_tags %}
 
 {% block content %}
-    {% stories_listing "stories" %}
+    {% featured_listing "featured" %}
```

Comment:
I realize now that with the way this tag is implemented we can just use the existing tag as is and just change the category. Please revert and just use `{% stories_listing "featured" %}`